### DISCUSSION TITLE
can't see the bar after installation

### DISCUSSION THREAD

**Discussion by caoguojun0** [FelixKratz/SketchyBar#431] [Q&A]
Hello
I have done everything as written in the instructions, also installed all dependencies, but still the bar does not appear, what could I be missing? thanks

**Comment by FelixKratz** [comment]
Make sure to copy the default configuration files to `~/.config/sketchybar` and make them executable with `chmod +x` by running the commands provided in the brew installation caveats section. Also make sure to hide the macOS menubar by following the instructions at https://felixkratz.github.io/SketchyBar/setup#hiding-the-original-macos-bar otherwise sketchybar might appear below the macOS bar.

Additionally, try running the bar directly from the commandline to see error or warning messages:
```bash
brew services stop sketchybar
sketchybar
```

If you are running yabai, make sure you have made room for sketchybar at the top via:
```bash
yabai -m config external_bar all:50:0
```

**Reply by marcelmanz** [reply]
~~Where are the default configuration files?~~

For anyone searching for the default configuration:
https://github.com/FelixKratz/SketchyBar/blob/master/sketchybarrc

**Comment by iRootPro** [comment]
I also have a similar problem. i have ventura, i disabled the standard menu bar. i added config and plugins run: `~/.config/sketchybar ❯ sketchybar` menu bar not show

I figured it out, yabai overlapped, it doesn't see that there is a menu there and just closes the menu bar, how can I customize the yabai + SketchyBar bundle?
